install.sh

```diff
@@ -53,6 +53,44 @@ printSuccess() {
   echo "${green}${msg[*]}${reset}"
 }
 
+checkJwt() {
+  local is_sandbox=$1
+  local php_path=$2
+  local environment="production"
+  if [[ $is_sandbox ]]; then environment="sandbox"; fi
+  if [[ -f pub/index.php ]]
+  then
+    # shellcheck disable=SC2016
+    $php_path -r '
```

Comment:
Adjust functional tests https://github.com/magento-swat/install-agent-helpers/actions/runs/2258601366

We need mock this method.
Maybe introduce a global environment variable INSTALL_AGENT_MODE=test

```
[[ "$INSTALL_AGENT_MODE" == "test" ]] && { echo "JWT Toeken - OK" ; exit 0 }
```